qhub/template/stages/07-kubernetes-services/modules/kubernetes/services/dask-gateway/variables.tf

```diff
@@ -31,8 +31,8 @@ variable "gateway-image" {
     tag  = string
   })
   default = {
-    name = "daskgateway/dask-gateway-server"
-    tag  = "0.9.0"
+    name = "ghcr.io/dask/dask-gateway-server"
```

Comment:
Hi @peytondmurray sorry for committing directly in the branch, I wanted to test a qhub deployment based on these changes, hope you don't mind. I also added the new version for gateway here, but I need to check if the controller is using the up-to-date version as well.

Reply:
No problem at all!